### ISSUE TITLE
release notes on swarm.ethereum.org

### ISSUE THREAD

**Issue by nonsense** [ethersphere/swarm#1400]
We don't have a place for our release notes that we attach to every release on our website.

Currently we only push binaries to the Downloads page, and add the release notes under a Reddit thread.

Release notes should be part of our website, or committed alongside the code in the `swarm/docs` folder?

**Comment by cobordism** [comment]
I think releases should be announced on our yet-to-be-created blog.

The website itself should be rather static unless and until we have dedicated comms people on our team who would take on the task of keeping it all up to date.

**Comment by xrn** [comment]
IMO If Swarm is moving to own repo - well-described github releases can be also enough good on the beginning

**Comment by acud** [comment]
i don't think we should have release notes in the website. should be communicated through github. we should just establish a `CHANGELOG` file in the base of the `swarm/` repo and append to it

**Comment by skylenet** [comment]
> i don't think we should have release notes in the website. should be communicated through github. we should just establish a `CHANGELOG` file in the base of the `swarm/` repo and append to it

**Some examples**: 
Terraform: https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md 
InfluxDB: https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md

**Comment by nonsense** [comment]
This ticket was created when we didn't have a repo ;) I like the CHANGELOG.md suggestion, let's do that. Rendering that on the website would be optional, same as the `docs` folder I guess.

**Comment by acud** [comment]
so can we close?

**Comment by nonsense** [comment]
No, let's close once we have the first release notes content in CHANGELOG.md and we establish the process of doing it.
